src/Http/BaseApplication.php

```diff
@@ -174,6 +174,11 @@ public function getPlugins(): PluginCollection
     public function bootstrap(): void
     {
         require_once $this->configDir . 'bootstrap.php';
+
+        $plugins = include $this->configDir . 'plugins.php';
```

Comment:
Can we make this an optional thing? That will make upgrading simpler as applications won't be broken when they upgrade and we can document how to adopt the 'new' thing.

I recognize that we're going into a major release and breaking changes are 'ok' but I feel that we should still try to make upgrading easier at little to no cost to us.

Reply:
Done